plottable.js

```diff
@@ -909,7 +909,7 @@ var Plottable;
 ///<reference path="../reference.ts" />
 var Plottable;
 (function (Plottable) {
-    Plottable.version = "1.10.0";
+    Plottable.version = "1.11.0";
```

Comment:
It doesn't make sense for this diff to occur, since either we are updating the version number of Plottable ("highly unlikely") or there is a potential merge with the develop branch that is not tracked via the commit / commit messages.  It is likely an issue with the built files.